src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -927,3 +927,42 @@ def test_storage_account_private_endpoint(self, storage_account):
             self.cmd('storage account private-endpoint-connection approve --account-name {sa} -g {rg} --name {sa_pec_name}')
 
         self.cmd('storage account private-endpoint-connection delete --id {sa_pec_id} -y')
+
+
+class StorageAccountSkuScenarioTest(ScenarioTest):
+    @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2019-04-01')
+    @ResourceGroupPreparer(name_prefix='clistorage', location='westus2')
+    @StorageAccountPreparer(name_prefix='clistoragesku', location='westus2', kind='StorageV2', sku='Standard_ZRS')
+    def test_storage_account_sku(self, resource_group, storage_account):
+        self.kwargs = {
+            'gzrs_sa': self.create_random_name(prefix='cligzrs', length=24),
+            'GZRS': 'Standard_GZRS',
+            'rg': resource_group,
+            'sa': storage_account
+        }
+        # Create storage account with GZRS
+        gzrs_name = self.create_random_name(prefix='cligzrs', length=24)
+        GZRS = 'Standard_GZRS'
+        #create_cmd = 'az storage account create -n {gzrs_sa} -g {rg} --sku {GZRS}'.format(gzrs_name, resource_group, GZRS)
```

Comment:
`gzrs_name = self.create_random_name(prefix='cligzrs', length=24)`
`GZRS = 'Standard_GZRS'`
Don't you comment out or delete the above two lines?

Reply:
removed!